b2b-sample/guardio-insurance-sample-application/components/editUser/editUserComponent.js

```diff
@@ -33,9 +33,16 @@ export default function EditUserComponent(props) {
 
     const [loadingDisplay, setLoadingDisplay] = useState(LOADING_DISPLAY_NONE);
 
-    const nameValidate = (name, errors) => {
-        if (!name) {
-            errors.name = 'This field cannot be empty'
+    const firstNameValidate = (firstName, errors) => {
```

Comment:
These validators can be generalized to a single validator.

Reply:
This is fixed with a later PR,

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/shared/util/util-front-end-util/src/lib/formUtil/formUtil.ts 
- https://github.com/wso2/samples-is/blob/09d8f7d6abe21d199f2c739e578b0883825d8cf2/b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.tsx#L60

### Commit
- https://github.com/wso2/samples-is/commit/9c330e205151b20f9436bf6aad9c4db85e216673#diff-2982d16cf307c73b8cbce49859f3040b0fa42f10c49acaa3952642cdd40632b5